yt/utilities/png_writer.py

```diff
@@ -1,19 +1,10 @@
 from io import BytesIO
 
-try:
-    # matplotlib switched from an internal submodule _png to using pillow (PIL)
-    # between v3.1.0 and v3.3.0
-    # So PIL should be available on any system where matplotlib._png doesn't exist
-    import matplotlib._png as _png
-except ImportError:
-    from PIL import Image
+from PIL import Image
```

Comment:
Just so we're on the same page -- if I'm reading it right, `PIL` should already be a transitive dependency, right?

Reply:
almost: it is a transitive dependency *if MPL 3.3 or newer is installed*. In the future, as we progressively fade out support to older versions, it will always be a transitive dependency, yes.